plugins/network-elements/nuage-vsp/src/com/cloud/network/resource/NuageVspResource.java

```diff
@@ -262,15 +260,19 @@ public String getName() {
     public PingCommand getCurrentStatus(long id) {
         if ((_relativePath == null) || (_relativePath.isEmpty()) || (_cmsUserInfo == null) || (_cmsUserInfo.length == 0)) {
             s_logger.error("Failed to ping to Nuage VSD");
+            _shouldAudit = true;
             return null;
         }
         try {
             login();
         } catch (Exception e) {
             s_logger.error("Failed to ping to Nuage VSD on " + _name + " as user " + _cmsUserInfo[1] + " Exception " + e.getMessage());
```

Comment:
Pass the exception instance into the ``error`` call to provide the stack trace in the log for debugging.

Also, what if cmsUserInfo is null or has a length less than 1?  Checks need to be added for these conditions to avoid an NPE or ``ArrayIndexOutOfBounds`` being thrown from this ``catch`` block.